core/bcast/bcast.go

```diff
@@ -158,18 +158,18 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty, pubkey core.
 		log.Debug(ctx, "V2 submit beacon committee subscriptions failed")
 
 		// Beacon node doesn't support v2 SubmitBeaconCommitteeSubscriptions endpoint (yet). Try with v1.
-		res, err := eth2exp.CalculateCommitteeSubscriptionResponse(ctx, b.eth2Cl, &sub.BeaconCommitteeSubscription)
+		isAggregator, err := eth2exp.IsAttestationAggregator(ctx, b.eth2Cl, sub.Slot, sub.CommitteeIndex, sub.SelectionProof)
```

Comment:
`eth2exp.CalculateCommitteeSubscriptionResponse` refactored to `eth2exp.IsAttestationAggregator` since there is no new type and only the IsAggregator field has to be updated.